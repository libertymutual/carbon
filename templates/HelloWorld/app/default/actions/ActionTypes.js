// APP ACTIONS

// AUTH ACTIONS
export const LOGIN = 'login';
export const LOGOUT = 'logout';
export const AUTH_UPDATE_AUTH_URL = 'auth_update_auth_url'
export const AUTH_UPDATE_CLIENT_ID = 'auth_update_client_id'
export const AUTH_UPDATE_CLIENT_SECRET = 'auth_update_client_secret'
export const AUTH_UPDATE_CONFIG_NAME = 'auth_update_config_name'
export const AUTH_UPDATE_PASSWORD = 'auth_update_password'
export const AUTH_UPDATE_USERNAME = 'auth_update_username'
export const AUTH_UPDATE_VALIDATOR_ID = 'auth_update_validator_id'

// COUNTER ACTIONS
export const INCREMENT = 'increment';
export const DECREMENT = 'decrement';

// BIOMETRIC ACTIONS
export const UPDATE_USERNAME = 'update_username';
export const UPDATE_PASSWORD = 'update_password';
export const SAVE_CREDENTIALS = 'save_credentials';
export const DELETE_CREDENTIALS = 'delete_credentials';
