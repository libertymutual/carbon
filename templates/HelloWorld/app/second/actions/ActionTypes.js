// APP ACTIONS

// HELLO WORLD ACTIONS
export const HELLO = 'hello';
export const GOODBYE = 'goodbye';

// COUNTER ACTIONS
export const INCREMENT = 'increment';
export const DECREMENT = 'decrement';
