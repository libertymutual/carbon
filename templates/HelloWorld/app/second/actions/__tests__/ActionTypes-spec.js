import * as ActionTypes from '../ActionTypes';

describe('ActionTypes tests', () => {

  test('HELLO action type returns hello', () => {
    expect(ActionTypes.HELLO).toMatchSnapshot();
  });

  test('GOODBYE action type returns goodbye', () => {
    expect(ActionTypes.GOODBYE).toMatchSnapshot();
  });


  test('INCREMENT action type returns increment', () => {
    expect(ActionTypes.INCREMENT).toMatchSnapshot();
  });

  test('DECREMENT action type returns decrement', () => {
    expect(ActionTypes.DECREMENT).toMatchSnapshot();
  });
});
