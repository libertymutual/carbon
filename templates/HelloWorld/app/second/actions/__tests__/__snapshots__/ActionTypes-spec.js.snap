// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes tests DECREMENT action type returns decrement 1`] = `"decrement"`;

exports[`ActionTypes tests GOODBYE action type returns goodbye 1`] = `"goodbye"`;

exports[`ActionTypes tests HELLO action type returns hello 1`] = `"hello"`;

exports[`ActionTypes tests INCREMENT action type returns increment 1`] = `"increment"`;
